src/chains-config/metadata-consts/index.ts

```diff
@@ -0,0 +1,45 @@
+import { Definition } from '../../types/chains-config';
+import { kusamaDefinitions } from './kusama-consts';
+import { polkadotDefinitions } from './polkadot-consts';
+
+/**
+ * Creates an object that orders each runtime to their appropriate data
+ *
+ * @param definitions An array of objects that group data based on runtimes
+ * and their extrinsicBaseWeight metadata
+ */
+const generateBlockWeightObject = (
+	definitions: Definition[]
+): Record<string, Definition> => {
+	const blockWeightObject = {};
+
+	for (let i = 0; i < definitions.length; i++) {
+		const runtimes: number[] = definitions[i].runtimes;
+
+		for (let p = 0; p < runtimes.length; p++) {
+			const version: number = runtimes[p];
+			blockWeightObject[version] = definitions[i];
```

Comment:
So I updated this, changed the naming and also did some abstraction with Definition. The reason it changed was because I also simplified the return value of `generateBlockWeightObject`. I think it also allows for more specific type's and is a lot easier to read. Let me know what you think.